src/jit/gentree.h

```diff
@@ -2334,10 +2334,85 @@ struct GenTreeColon: public GenTreeOp
         {}
 };
 
-/* gtCall   -- method call      (GT_CALL) */
+// gtCall   -- method call      (GT_CALL)
 typedef class fgArgInfo *  fgArgInfoPtr;
 enum class InlineObservation;
 
+// Multi-reg return type descriptor of a GT_CALL node.
+// x64 Unix, Arm64, Arm32 and x86 allow a value to be returned in multiple
```

Comment:
This descriptor is initialized only for struct returns.  That struct could be returned in zero, one or multiple registers.  But this descriptor is needed only when a type is returned in multiple registers.  In a single register case struct type will be replaced with one of the scalar types.  For now name is chosen to be for its intended purpose.